### DISCUSSION TITLE
What should we do about this?

### DISCUSSION THREAD

**Discussion by catfoolyou** [3kh0/ext-remover#1156] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
It appears that the guy who made the EaglerX 1.19 server is now bent on screwing us over
https://github.com/CaenJones/ChromeSEC
How bypassable are the things that he talks about in that repo?

My conversation with him:
https://github.com/CaenJones/ChromeSEC/issues/1

**Comment by HelloIloveGimkit2** [comment]
just mass report the repo

**Reply by HelloIloveGimkit2** [reply]
or mass report his user

**Reply by catfoolyou** [reply]
For what though?

**Reply by HelloIloveGimkit2** [reply]
idk for like stealing our work just come up with a fake reason

**Reply by catfoolyou** [reply]
Can't do that.

**Reply by HelloIloveGimkit2** [reply]
idk just try and blackmail him by saying smth like:

"We will report your game to people that way people don't play it" smth along those lines.

**Reply by catfoolyou** [reply]
That's just dumb

**Reply by s0urce-c0de** [reply, marked as answer]
Stealing buisness from chrome os support

**Reply by catfoolyou** [reply]
Real

**Reply by s0urce-c0de** [reply]
how can we report him though?

**Reply by ghost** [reply]
what is he doing exactly?

**Comment by ghost** [comment]
Give me a shoutout in EXT-Remover frfr

**Reply by ghost** [reply]
also props to my friend for telling me about this thread xD

**Reply by catfoolyou** [reply]
CRSH2TTY is gonna ruin your business lmfao

**Reply by ghost** [reply]
I think I already found a fix

https://support.google.com/chrome/a/answer/6352858?hl=en

**Reply by ghost** [reply]
and that's what people said to me about Sh1mmer :P 

![image](https://github.com/user-attachments/assets/3e94f0dd-8075-43e8-93eb-3d11abb45a70)

**Reply by s0urce-c0de** [reply]
and how exactly would it work? and how do you plan to stop me from getting a job and buying an exact copy of the model my school has? how would they know?

**Reply by ghost** [reply]
the district could just remove enrollment perms, and Google admin console has unique device identifier for each system

**Reply by s0urce-c0de** [reply]
FUN! no fwmp

**Reply by catfoolyou** [reply]
> I think I already found a fix
> 
> https://support.google.com/chrome/a/answer/6352858?hl=en

That doesnt fix it iirc

**Reply by ghost** [reply]
Honestly, I don't really know. The information I have gotten from Google makes it seem like it would work, and other people that I talked to make it seem like forced re-enrollment was not active when they conducted the exploit.

**Reply by ghost** [reply]
> FUN! no fwmp

Account perms for enrollment are cloud-based and not local. Same goes for device identifiers and policy sync dates shown in Google admin.

**Reply by catfoolyou** [reply]
hehehehaw

**Reply by ghost** [reply]
wydm?

**Reply by s0urce-c0de** [reply]
how do you plan to patch the pencil method

**Reply by ghost** [reply]
you don't need to patch it, you just have to detect it.

**Reply by s0urce-c0de** [reply]
i could say the same for all the other exploits. so why are you making patches for those? it is our right to goguardian not knowing my physical location (i wonder why...), my SYSTEM INFORMATION, my identity, alarms, EVERY PAGE I VISIT, etc.
![image](https://github.com/user-attachments/assets/f6851b43-93a8-4008-a755-bf1cbe2f898a)

**Reply by ghost** [reply]
The school district you went to is legally obligated to have filtering and monitoring tools and really, there are only 5-6 "choices" with the same amount of data collection. The vendors signed agreements with the district so they cant share data with advertisers. Also, its not your computer in the end so you have no right to privacy on it.

**Reply by ghost** [reply]
However, if you want to blame someone it should be the private companies and not the school districts that in the end are trying to serve students.

**Reply by ghost** [reply]
Especially since most controls and protections are locked behind paywalls or higher tiers, districts deserve something like chromeSEC in order to properly protect students without having to pay more.

They are doing the best they can with what they have

**Reply by catfoolyou** [reply]
"Protect students" is the most authoritarian thing I've ever heard. Holy shit. 

We do have the right to privacy, even if the districts signed off said rights to some shitty corporation.

The companies that develop these things are complete motherfuckers. They lobby laws and regulations that force districts to use their shitty services and pay them, squeezing the maximum amount of profit possible from the situation. Fuck 'em.

**Reply by ghost** [reply]
Could you think of a situation where something like lightspeed filter or lightspeed alert could be useful and helpful for the student? Im really asking

**Reply by s0urce-c0de** [reply]
> Could you think of a situation where something like lightspeed filter or lightspeed alert could be useful and helpful for the student? Im really asking

no

**Reply by s0urce-c0de** [reply]
if you want , you can provide an example :)

**Reply by s0urce-c0de** [reply]
@CaenJones Look, if you want to continue arguing we can do it on discord, my discord: [@s0urce_c0de](https://discord.com/users/947649923342540860). Also goguardian if very stupid :skull:. `https://github.com/3kh0/ext-remover` is blocked, but `https://github.com/3kh0/ext-remover/` is not (see, just add a slash!) Also, if you may or may not be debugging goguardian you can find the entire filter list, get the highest priority whitelist and use that to bypass it. And I can use a netlog from 2 years ago to do this, they don't expire authorization.

**Reply by S-PScripts** [reply]
i know delete this repo and make a new one sex

**Reply by ghost** [reply]
> i know delete this repo and make a new one sex

@S-PScripts do you want to join the discussion on discord?

**Reply by ghost** [reply]
> @CaenJones Look, if you want to continue arguing we can do it on discord, my discord: [@s0urce_c0de](https://discord.com/users/947649923342540860). Also goguardian if very stupid 💀. `https://github.com/3kh0/ext-remover` is blocked, but `https://github.com/3kh0/ext-remover/` is not (see, just add a slash!) Also, if you may or may not be debugging goguardian you can find the entire filter list, get the highest priority whitelist and use that to bypass it. And I can use a netlog from 2 years ago to do this, they don't expire authorization.

We started on discord for those following this thread. Also, no I  have not and never will debug goguardian :P 

and their website looks bad

**Reply by ghost** [reply]
we should also invite @3kh0 frfr

**Reply by 3kh0** [reply]
> i could say the same for all the other exploits. so why are you making patches for those? it is our right to goguardian not knowing my physical location (i wonder why...), my SYSTEM INFORMATION, my identity, alarms, EVERY PAGE I VISIT, etc. @s0urce-c0de 

Your first mistake is expecting privacy on your school-issued device. Yes, it is pretty creepy why it would need all those permissions, but remember this is not your device. You do not own it. You are simply renting it from your school district for the school year. Bring your own device. I have been doing that, and it works. If your school does not allow you to BOYD, then whomp whomp, but expecting to have complete privacy on a device you do not own is absurd.

**Reply by s0urce-c0de** [reply]
> > i could say the same for all the other exploits. so why are you making patches for those? it is our right to goguardian not knowing my physical location (i wonder why...), my SYSTEM INFORMATION, my identity, alarms, EVERY PAGE I VISIT, etc.
> 
> Your first mistake is expecting privacy on your school-issued device. Yes, it is pretty creepy why it would need all those permissions, but remember this is not your device. You do not own it. You are simply renting it from your school district for the school year. Bring your own device. I have been doing that, and it works. If your school does not allow you to BOYD, then whomp whomp, but expecting to have complete privacy on a device you do not own is absurd.

no need for all the pings, you misspelled BYOD, yes I expect privacy of goguardian not selling my geolocation, and i am already following all threads i comment on, no need to ping me.

**Reply by S-PScripts** [reply]
> > i know delete this repo and make a new one sex
> 
> @S-PScripts do you want to join the discussion on discord?

where?

**Reply by S-PScripts** [reply]
> > > i could say the same for all the other exploits. so why are you making patches for those? it is our right to goguardian not knowing my physical location (i wonder why...), my SYSTEM INFORMATION, my identity, alarms, EVERY PAGE I VISIT, etc.
> > 
> > 
> > Your first mistake is expecting privacy on your school-issued device. Yes, it is pretty creepy why it would need all those permissions, but remember this is not your device. You do not own it. You are simply renting it from your school district for the school year. Bring your own device. I have been doing that, and it works. If your school does not allow you to BOYD, then whomp whomp, but expecting to have complete privacy on a device you do not own is absurd.
> 
> no need for all the pings, you misspelled BYOD, yes I expect privacy of goguardian not selling my geolocation, and i am already following all threads i comment on, no need to ping me.

you misspelled boyd 🤓 ☝️

**Reply by s0urce-c0de** [reply]
:|

**Reply by nathblade16** [reply]
> > i could say the same for all the other exploits. so why are you making patches for those? it is our right to goguardian not knowing my physical location (i wonder why...), my SYSTEM INFORMATION, my identity, alarms, EVERY PAGE I VISIT, etc. @s0urce-c0de
> 
> Your first mistake is expecting privacy on your school-issued device. Yes, it is pretty creepy why it would need all those permissions, but remember this is not your device. You do not own it. You are simply renting it from your school district for the school year. Bring your own device. I have been doing that, and it works. If your school does not allow you to BOYD, then whomp whomp, but expecting to have complete privacy on a device you do not own is absurd.

they don't even give us privacy on byod devices

**Comment by 3kh0** [comment]
I saw this coming: some kid is using ChatGPT to find ways to help sysadmins. 

For most exploits, the publisher commonly makes a write up about how the exploit works (see LTMEAT, sh1mmer, and more), making the sysadmin's job easier than ever. It takes a very minimal technical understanding to read through these.

This has already been done too, sysadmins commonly do talk to other sysadmins and share what works and what does not (see https://www.reddit.com/r/k12sysadmin/comments/52eu3r/chromebook_policy_checklist/) and you can jsut browse the r/k12sysadmin subreddit to see more posts of people offering guidance on the latest exploits

You are wild for thinking that all sysadmins are stupid and do not know how to work their systems. This might be true for some cases, but not for most school districts. Instead of helping them, you are just talking down to them like they are children. No one is going to actually use your guide for what you think they will; they will just see it as disrespect.

Please learn how to properly talk to adults.

(pinging just so you get the notification)
@catfoolyou @CaenJones

**Reply by ghost** [reply]
Hi @3kh0! It's nice to hear from you! I did not think you would respond.

1. To be perfectly honest, I had tried asking Google admin/policy questions to ChatGPT while researching for chromeSEC. In the end, I always got terrible responses and no real answers. I wish I were able to use it though! It would have made the writing time much shorter. 

2. I do not doubt in my mind that admins (especially at schools) know what they are doing, and understand Google Admin. The issue is, most important group policies are left off by default, and Google's own documentation is normally so obscure it rarely delivers detailed instructions on what something is / how to fix it. (why 3 policy lists? 😢) While writing/researching stuff for chromeSEC, I often had to go to the far ends of the internet to find a random post talking about how to implement something. As someone who has been around and worked with school sysadmins (multi-district now!), they do not have the time/energy to go about hunting for ways to harden their Chromebooks. They deserve better than for-profit Google support and dead-end searches. Hence the creation of an all-inclusive guide.

3. Again, while I'm sure some people do look at writeups on Discord and Github it's not the majority. Admins have better things to do than go poking around their Chrome policy when they could be responding to tickets or helping students. You are generously overestimating their pool of free time and attention they give you. 

Have a good day!

**Reply by funkymokey** [reply]
never give up nanana

**Reply by S-PScripts** [reply]
e
